.gitignore

```diff
@@ -11,3 +11,6 @@ snapshots
 build
 dist
 CloudFerry.egg-info
+configuration.ini
+dump.sql
```

Comment:
IIRC the MysqlDump task will place a copy of the dst cloud database in your current working directory.  If one executes a fabric task including this task from the root of the repo, this file will appear, but should be ignored by the git repo.

(Unless I'm doing it wrong)